packages/core/src/cli/index.ts

```diff
@@ -22,6 +25,35 @@ const argv = yargs
     boolean: true,
     description: 'Whether to emit declaration files',
   })
+  .option('build', {
+    alias: 'b',
+    boolean: true,
+    description:
+      'Build one or more projects and their dependencies, if out of date. Same as the TS `--build` flag.',
+    // As with TS itself, we *must* emit declarations when in build mode
+    conflicts: 'declaration',
+  })
+  .option('clean', {
+    implies: 'build',
+    boolean: true,
+    description: 'Delete the outputs of all projects.',
+    conflicts: 'watch',
+  })
+  .option('force', {
+    implies: 'build',
+    description: 'Act as if all projects are out of date. Same as the TS `--force` flag.',
+    type: 'boolean',
+  })
+  .option('dry', {
+    implies: 'build',
+    description: `Show what would be built (or deleted, if specified with '--clean'). Same as the TS \`--dry\` flag.`,
+    type: 'boolean',
+  })
+  .option('incremental', {
+    description:
+      'Save .tsbuildinfo files to allow for incremental compilation of projects. Same as the TS `--incremental` flag.',
+    type: 'boolean',
+  })
```

Comment:
Should we drop these flags (or maybe comment them out) until we support them?

Reply:
Commenting them out till supported seems like the right move, will do.

Reply:
4d9db80